### DISCUSSION TITLE
How to make github pages display a gulp build?

### DISCUSSION THREAD

**Discussion by perfectiones** [community/community#35914] [Other Feature Feedback, Questions, & Ideas]
Hello, I have repositorie with GULP. I hosted it on GITHUB Pages, but site show only site name.
Please help me with this problem.
*Branch is not a problem

My repository - https://github.com/perfectiones/BlueCollar.git

**Comment by MitkoVtori** [comment]
It's because you have only README.md in the repo. To run the website you need to add files and your main page should be named index.
example:
![Screenshot 2022-10-12 192048](https://user-images.githubusercontent.com/112943652/195396370-a770a746-2c8d-4809-a7e4-bc920916fcc3.png)
I hope this help!

### If yes, you can mark this as answered so everybody with the same problem will have solution.

**Reply by MitkoVtori** [reply]
Also, I just saw this:
![Screenshot 2022-10-12 192248](https://user-images.githubusercontent.com/112943652/195396913-70dddd1f-9963-4898-adcd-3c4eb5951924.png)
This is your master branch but it isn't the default branch. Also you have failed check. Try one of this three things:
change the website main branch
fix the issue
create a pull request from master to main
